python/taichi/lang/__init__.py

```diff
@@ -453,18 +453,15 @@ def check_version():
         req = request.Request('https://metadata.taichi.graphics/check_version',
                               method='POST')
         req.add_header('Content-Type', 'application/json')
-        with request.urlopen(req, data=payload, timeout=3) as response:
+        with request.urlopen(req, data=payload, timeout=5) as response:
             response = json.loads(response.read().decode('utf-8'))
             if response['status'] == 1:
-                print(
-                    f'Your Taichi version {version} is outdated. The latest version is {response["latest_version"]}, you can use\n'
-                    + f'pip install taichi=={response["latest_version"]}\n' +
-                    'to upgrade to the latest Taichi!')
-            elif response['status'] == 0:
-                # Status 0 means that user already have the latest Taichi. The message here prompts this infomation to users.
-                print(response['message'])
-    except Exception as error:
-        print('Checking lastest version failed:', error)
+                version_path = os.path.join(_ti_core.get_repo_dir(),
+                                            'latest_version')
+                with open(version_path, 'w') as f:
+                    f.write(response['latest_version'])
+    except Exception:
```

Comment:
Wild but might be a good fit here as we discussed @rexwangcc @strongoier :P  : 
```
except:
    pass
```

Reply:
OK, didn't know it can work like this

Reply:
didn't know it either, tested with:

```python
try:
     raise BaseException(99)
except:
     print(42)
```